layouts/calendar_layout.py

```diff
@@ -0,0 +1,88 @@
+import dash_html_components as html
+import layouts.style.style as style
+from tools.calendar_generation import calendar_config
+import dash_core_components as dcc
+
+def change_calendar_content():
+    """
+    Create layout for calendar
+
+    :return: Layout with 1 second refresh interval
+    :rtype: dash.development.base_component.ComponentMeta
+    """
+    #calendarID = calendar_config()
+    result = calendar_config()
+    return html.Div([
+        dcc.ConfirmDialog(
+                         id='confirm-good',
+                         message='Event added',
+        ),
+        dcc.ConfirmDialog(
+                         id='confirm-bad',
+                         message='There was an error during adding an event. Please check server logs for an error.',
+        ),
+
+        html.Iframe(
+                   src=result[0],
```

Comment:
What if `config.yml` does not exist?  Then result will be None, because calendar_config will not return result.